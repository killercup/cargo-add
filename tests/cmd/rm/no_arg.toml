bin.name = "cargo-rm"
args = ["rm"]
status.code = 2
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
