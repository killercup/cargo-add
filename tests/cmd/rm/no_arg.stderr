error: The following required arguments were not provided:
    <CRATE>...

USAGE:
    cargo rm [OPTIONS] <CRATE>...

For more information try --help
